mycore-acl/src/main/java/org/mycore/mcr/acl/accesskey/restapi/v2/access/MCRAccessKeyRestAccessCheckStrategy.java

```diff
@@ -97,13 +99,18 @@ private void handleGetRequest(ContainerRequestContext requestContext) {
             ensureHasManagePermission(accessKeyDto);
         } else {
             final MultivaluedMap<String, String> queryParameters = requestContext.getUriInfo().getQueryParameters();
-            final List<String> permissions = queryParameters.get(MCRAccessKeyRestConstants.QUERY_PARAM_PERMISSIONS);
-            if (permissions == null || permissions.isEmpty()) {
-                throw new ForbiddenException("permission is required");
+            final String permissionsParam = queryParameters.getFirst(MCRAccessKeyRestConstants.QUERY_PARAM_PERMISSIONS);
+            if (permissionsParam == null || permissionsParam.trim().isEmpty()) {
```

Comment:
`permissionsParam.isBlank()` should be used instead of `permissionsParam.trim().isEmpty()`